Tests/SourceGenerators/SourceGeneratorAssemblyToProcess/Generated/Realm.SourceGenerator/Realms.SourceGenerator.RealmGenerator/Person_generated.cs

```diff
@@ -0,0 +1,308 @@
+﻿// <auto-generated />
+using SourceGeneratorPlayground;
+using SourceGeneratorPlayground.Generated;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Runtime.Serialization;
+using System.Xml.Serialization;
+using System.Reflection;
+using System.ComponentModel;
+using Realms;
+using Realms.Weaving;
+using Realms.Schema;
+
+namespace SourceGeneratorPlayground
+{
+    [Generated]
+    [Woven(typeof(PersonObjectHelper))]
+    public partial class Person : IRealmObject, INotifyPropertyChanged, IReflectableType
+    {
+        public static ObjectSchema RealmSchema = new ObjectSchema.Builder("Person", ObjectSchema.ObjectType.RealmObject)
+        {
+            Property.Primitive("Id", RealmValueType.Guid, isPrimaryKey: true, isIndexed: false, isNullable: false, managedName: "Id"),
```

Comment:
Do we need to specify `managedName` when it's the same as the Core name?

Reply:
It's not necessary, but I think it doesn't hurt, and we avoid complicating the generator code.